client/state/login/actions.js

```diff
@@ -127,3 +128,28 @@ export const loginUserWithTwoFactorVerificationCode = ( user_id, two_step_code,
 			return Promise.reject( errorMessage );
 		} );
 };
+
+/**
+ * Sends a two factor authentication recovery code to the given user.
+ *
+ * @param  {Number}    userId        Id of the user trying to log in.
+ * @param  {String}    twoStepNonce  Nonce generated for verification code submission.
+ * @return {Function}                Action thunk to trigger the request.
+ */
+export const sendSmsCode = ( userId, twoStepNonce ) => dispatch => {
+	return request.post( 'https://wordpress.com/wp-login.php?action=send-sms-code-endpoint' )
```

Comment:
We should either add another config entry similar to `config( 'two_step_authentication_xhr' )` or extract the url and provide different action names for each method specified in this file.